components/EventDetail.js

```diff
@@ -281,6 +293,11 @@ export default function EventDetail({ event = {} }) {
             </Button>
           </>
         )}
+        {session?.id && (
+          <Button onClick={handleToggleAttending}>
+            {isAttending ? "Won't attend" : "Attend"}
+          </Button>
+        )}
```

Comment:
You could think about making this a component, since you use it more than once ;)

Reply:
@julialoeschel The Button Component is already a custom Component.